src/rust/plugin-work-queue/sqlx-data.json

```diff
@@ -0,0 +1,202 @@
+{
```

Comment:
Since this is a generated file, you may want to add it to `.gitattributes`

Reply:
Neat, will do.